css/style/card.css

```diff
@@ -0,0 +1,91 @@
+.card{
+   position: relative;
+    background-color: var( --bg-secondary);
+    border-radius: 10px;
+    display:inline-block;
+}
+.simple-card{
+    max-width:300px;
+    min-height:280px;
+}
+.text-only-card{
+    max-width:300px;
+    min-height:150px;
+}
+.horizontal-card{
+    max-width:320px;
+    min-height:180px;
+}
+.card-img{
+    border-top-left-radius: 10px;
+    border-top-right-radius: 10px;
+    width:300px;
+    height:180px;
+    z-index:9;
+}
+.card-heading{
+    font-weight: 900;
+    color:var(--vibrant);  
+}
+.card-textarea{
+    display:flex;
+    justify-content: space-between;
+}
+.card-icon{
+    margin-top:5px;
+}
+.card-icon:hover{
+    font-weight: 800;
+    color:var(--vibrant);  
+}
+.dismiss-card-btn{
+    position: absolute;
+    top:10px;
+    right:10px;
+    opacity: .8;
+}
+.dismiss-card-btn:hover{
+    opacity: 1;
+    font-weight: 8000;
+}
+.card-img-overlay{
+    width:300px;
+    height:200px;
+    border-top-left-radius: 10px;
+    border-top-right-radius: 10px;
+}
+
+.card-img-overlay:hover {
+    opacity:0.2;
+    /* filter: grayscale(100); */
+}
+.card-img-overlay:hover + .overlay-card-content{
+   
+    display:block;
```

Comment:
keep consistent spacing, else everything looks good to me

Reply:
yeah, Tha's by mistake correcting it. Thank you for reviewing :)